common/scripts/configs/credentials.template

```diff
@@ -0,0 +1,3 @@
+[default]
+aws_access_key_id={{aws_access_key}}
```

Comment:
lets rename the template variable to `{{AWS_ACCESS_KEY}}` to be consistent with variables in vault config template